src/index.html

```diff
@@ -1,12 +1,47 @@
 <!DOCTYPE html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day--qwe calendar--month-length--27">
```

Comment:
```suggestion
    <div class="calendar calendar--start-day--wed calendar--month-length--28">
```
Чому qwe і 27? це ж не валідні значення для цих модифікаторів